Added last toggled item (#983)
vtex/styleguide: CHANGELOG.md

## [Unreleased]

### Added

> - Last toggled item into `EXPERIMENTAL_useCheckboxTree` to be returned by the `onToggle` function.

Reviewer: ```suggestion
- Last toggled item into `EXPERIMENTAL_useCheckboxTree` to be sent as param to `onToggle` function.
```
